FishBun/src/main/java/com/sangcomz/fishbun/FishBun.kt

```diff
@@ -0,0 +1,30 @@
+package com.sangcomz.fishbun
+
+import android.app.Activity
+import androidx.fragment.app.Fragment
+import com.sangcomz.fishbun.adapter.image.ImageAdapter
+import java.lang.ref.WeakReference
+
+class FishBun(context: Activity?) {
```

Comment:
1. It's mistake declaring of private constructor, thank you. apply in https://github.com/sangcomz/FishBun/pull/175/commits/0cff586b8381e729ec2f765b87e4d0ed20123d20


2. `fragment.getActivity()` returns nullable. So It can be solved As
```kotlin
class FishBun(context: Activity) {
    ...
    companion object {
        @JvmStatic
        fun with(fragment: Fragment) = FishBun(fragment.activity!!) << force safely
    }
}
```
But it doesn't handle null exception. So i handled exception in `startAlbum()` method. As
``` kotlin
override fun startAlbum() =
    fishBun.context?.let { context ->
        ...
    } ?: throw NullPointerException("Activity or Fragment Null")
```
If activity or fragment context is not `WeakReference`, don't need to care that.